python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -194,7 +195,7 @@ def dictToTuple(dict_, keys):
     return tuple(dict_[k] for k in keys)
 
 
-def getCcdIdListFromExposures(expRefList, level="sensor", ccdKeys=["ccd"]):
+def getCcdIdListFromExposures(expRefList, level="sensor", ccdKeys=["ccd"], visitKeys=["visit"]):
```

Comment:
Why is this added as an argument?

Reply:
Good catch. It's vestigial, from before I made `checkCcdIdLists` a method instead of a free function.